scripts/migrations/10

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Migration 10 --> 11
+
+# Force reconcile hr keycloak-configure
+kubectl delete po -l app=source-controller -n cozy-fluxcd
+kubectl annotate helmrelease keycloak-configure -n cozy-keycloak \
+  reconcile.fluxcd.io/forceAt="$(date --rfc-3339=ns)" \
+  reconcile.fluxcd.io/requestedAt="$(date --rfc-3339=ns)" \
+  --overwrite
```

Comment:
```suggestion
timestamp=$(date --iso-8601=ns)
kubectl delete po -l app=source-controller -n cozy-fluxcd
kubectl annotate helmrelease keycloak-configure -n cozy-keycloak \
  reconcile.fluxcd.io/forceAt="$timestamp" \
  reconcile.fluxcd.io/requestedAt="$timestamp" \
  --overwrite
```

Flux's format of these annotations looks more like ISO-8601, also call `date` once and set it to this value for both annotations.